elements/urth-help/urth-help.html

```diff
@@ -0,0 +1,27 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+Creates a Polymer element that displays urth elements documentation page.
+-->
+<dom-module id="urth-help">
+
+    <template>
+        <iframe src="/nbextensions/urth_widgets/docs/docs.html" style="width:100%; height: 300px;">
```

Comment:
I don't think the url `/nbextensions/urth_widgets/docs/docs.html` will always be valid. This assumes the notebook is always installed at the root which may not be true.